### PR TITLE
Add an Atom feed

### PR DESCRIPTION
**Pull request by j-f1**
This lets people follow the latest updates from their feed reader

### CODE REVIEW

**Comment by ineffyble** [comment]
I'm flattered that anyone would want a feed.

I am a little perturbed that we're having to handcraft XML though. I guess it looks less bad when you're using a templating language like nunjucks vs JS string interpolation.

**Comment by ineffyble** [comment]
Do you reckon we need the RSS feed linked? Having the meta tag should mean it gets picked up automatically by readers, and it takes up more footer space. Might want to hide the link on mobile at least where it causes the footer to become taller.

**Comment by j-f1** [comment]
> I am a little perturbed that we're having to handcraft XML though. I guess it looks less bad when you're using a templating language like nunjucks vs JS string interpolation.

Yeah. The alternatives are to pick up some off the shelf package to generate the XML (which means the existing page objects would have to be remapped to the API that package exposes), switch back to Nunjucks (which the example in the docs uses, but doesn’t work properly with async filters/shortcodes), or output a JSON feed instead (easier to produce but somewhat less compatible)

**Comment by j-f1** [comment]
> Do you reckon we need the RSS feed linked?

I put the link there as a way to educate folks new to RSS feeds but I think it’s totally reasonable to get rid of the link.

**Comment by ineffyble** [comment]
Thank you! Might make more tweaks but can do that later.

File: feed.11ty.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ exports.render = async function ({ permalink, collections }) {
     <?xml version="1.0" encoding="utf-8" ?>
     <feed xmlns="http://www.w3.org/2005/Atom" xml:base="${metadata.url}">
       <title>${metadata.title}</title>
-      <link href="${permalink | rss.absoluteUrl(metadata.url)}" rel="self"/>
-      <link href="${metadata.url}"/>
+      <link href="${permalink || rss.absoluteUrl(metadata.url)}" rel="self"/>
```
Comment (j-f1) [inline feed.11ty.js:26]: Oops! Looks like I missed this one when converting from Nunjucks. It should actually be:

```js
rss.absoluteUrl(permalink, metadata.url)
```

Comment (ineffyble) [reply feed.11ty.js:26]: We're providing the `xml:base` so having a relative path in the `href` appears to be fully valid. But that does explain it, I was a little confused.

**Pull request merged**